client/src/components/profile/components/TimelinePagination.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TimelinePagination = props => {
+  const { pageNo, totalPages, firstPage, prevPage, nextPage, lastPage } = props;
+  return (
+    <nav
+      aria-label='Timeline Pagination Navigation'
+      className='timeline-pagination'
+      role='navigation'
+    >
+      <ul className='timeline-pagination_list'>
+        {totalPages > 10 && (
+          <li
+            className='timeline-pagination_list_item'
+            style={{
+              visibility: pageNo === 1 ? 'hidden' : 'unset'
+            }}
+          >
+            <button
+              aria-label='Goto First page'
```

Comment:
```suggestion
              aria-label='Go to first page'
```
Goto is just a computing term, as far as I know.  Presumably it would be pronounced differently from 'go to' by screen readers.